CHANGELOG.rst

```diff
@@ -31,6 +31,11 @@ Fixed
 - Fixed bug that stopped Dockerfiles from building version 1.4.4.
 - Fixed format guessing for e2e stories with intent restated as ``/intent``
 
+Changed
```

Comment:
there were some releases in between. you need to update the location in the changelog

Reply:
can you see if my recent update is right?  If not we can talk about it tomorrow on our call to ensure I have the changelog right.